.claude/skills/cut-release/SKILL.md

```
@@ -0,0 +1,60 @@
+  git tag vX.Y.Z $(git rev-parse origin/main)
+  git push origin vX.Y.Z
+  ```
+- Or have them use the GitHub web UI: Releases → Draft a new release → Choose a tag → type `vX.Y.Z` → Create new tag on publish.
+
+After they say it's done, **verify independently** — `mcp__github__get_tag` or `list_tags` — before proceeding; don't trust "done" at face value, the push can silently fail or land on the wrong ref, and GitHub's API can lag a few seconds behind a fresh push.
+
+## 4. Apply curated notes
+
+The tag push alone triggers `release.yml` and creates a release with GitHub's auto-generated notes (flat, ungrouped, per `.github/release.yml`'s wildcard rule) — that's expected, not a bug. To replace it with the categorized draft from step 2:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)